staging/src/k8s.io/kubectl/go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/exponent-io/jsonpath v0.0.0-20151013193312-d6023ce2651d
 	github.com/fatih/camelcase v1.0.0
 	github.com/fvbommel/sortorder v1.0.1
+	github.com/go-task/slim-sprig v0.0.0-20210107165309-348f09dbbbc0
```

Comment:
new dep to kubectl, but not to the kubernetes project. would be very useful to make it available to templates